src/app/components/feed/SaveFeed.js

```diff
@@ -594,6 +619,27 @@ const SaveFeed = (props) => {
         onCancel={() => { setShowConfirmationDialog(false); }}
         isSaving={saving}
       />
+
+      {
+        isEditing &&
+          <NavigateAwayDialog
+            hasUnsavedChanges
+            title={
+              <FormattedMessage
+                id="tasks.confirmLeaveTitle"
+                defaultMessage="Do you want to leave without saving?"
+                description="This is a prompt that appears when a user tries to exit a page before saving their work."
+              />
+            }
+            body={
+              <FormattedMessage
+                id="tasks.confirmLeave"
```

Comment:
Message ids should follow this pattern (for now): `componentName.descriptiveId` please replace `tasks` with `saveFeed`

Reply:
Ah, oops! Good catch, updated!